test/runtests.jl

```diff
@@ -203,6 +212,10 @@ module Namespace91
         @test A91V1(; a=1) == A91V1(; a=1)
         @test isequal(A91V1(; a=1), A91V1(; a=1))
         @test read(tobuffer([A91V1(; a=1)], A91V1SchemaVersion())).a == [1]
+
+        a = A91V1(; a=1)
+        a2 = @set a.a = 2
+        @test a2.a == 2
```

Comment:
Why are these tests needed? Seems like the later testset covers this. At the very least this should be made into a separate testset inside this module

Reply:
just to make sure that the ConstructionBase name is interpolated correctly as well, since we generate code with `$ConstructionBase.` in teh macro

Reply:
Okay, would have been nice to have this as a separate testcase inside the submodule as mentioned. I'll leave this unresolved